src/test/java/uk/gov/digital/ho/hocs/casework/api/CaseDataServiceTest.java

```diff
@@ -899,42 +885,6 @@ public void shouldNotDeleteCaseMissingCaseUUID() {
         verifyNoMoreInteractions(caseDataRepository);
     }
 
-    @Test
```

Comment:
These have been removed from CaseDataServiceTest and moved to CaseDataTypeServiceTest